examples/vault/README.md

```diff
@@ -0,0 +1,323 @@
+# Trusted Service Identity Demo with Vault
+This demo uses the [TSI Vault authentication plugin](../../components/vault-plugin/README.md)
```

Comment:
nit:  replace all the relative links with absolute? `/components/vault-plugin/README.md`. What do you think?